boards/arm/dragonclaw/dragonclaw.dts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2022 Google Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <st/f4/stm32f412Xg.dtsi>
+#include <st/f4/stm32f412z(e-g)tx-pinctrl.dtsi>
+
+/ {
+	model = "Google Dragonclaw development board";
+	compatible = "google,dragonclaw-fpmcu", "st,stm32f412";
+
+	chosen {
+		zephyr,console = &usart2;
+		zephyr,shell-uart = &usart2;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,flash-controller = &flash;
+	};
+};
+
+&clk_hsi {
+	/* HSI clock frequency is 16Mhz */
+	status = "okay";
+};
+
+&clk_lsi {
+	/* LSI clock frequency is 32768khz */
+	status = "okay";
+};
+
+&pll {
+	div-m = <8>;
+	mul-n = <192>; /* 16Mhz * 192/8 = 384Mhz VCO clock */
+	div-p = <4>; /* 96Mhz PLL general clock output */
+	div-q = <8>; /* 48Mhz PLL output for USB, SDIO, RNG */
+	clocks = <&clk_hsi>;
+	status = "okay";
+};
+
+&rcc {
+	clocks = <&pll>; /* Select PLL as SYSCLK source (96Mhz) */
+	ahb-prescaler = <1>; /* SYSCLK not divided */
+	clock-frequency = <DT_FREQ_M(96)>; /* AHB frequency */
+	apb1-prescaler = <2>; /* AHB clock divided by 2 */
+	apb2-prescaler = <2>; /* AHB clock divided by 2 */
+};
+
+/* USART1: PA9/PA10 (TX/RX) to AP (Host Commands and MKBP) */
```

Comment:
More of a personal preference, I would drop the `USART1: PA9/PA10 (TX/RX)` and just write something like `AP UART (Host Commands and MKBP)`. Reasoning is: the information is immediately available in the lines below and that's the actual source of truth for the configuration, duplicating it in the comment adds one more place to keep in sync manually, may end up copypasted wrongly in a future board variant and make the whole thing confusing. Same for the other node comments.